### PR TITLE
update to 2.14.1

### PR DESCRIPTION
**Pull request by Arishamays1**

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -45,6 +45,7 @@ export CFLAGS="${CFLAGS} -fno-merge-constants"
 
 make -j${CPU_COUNT} ${VERBOSE_AT}
 make install
+alias touch=gtouch
```
Comment (marcoesters) [inline recipe/build.sh]: Why was this added? `touch` is a standard unix command. We shouldn't need to alias it to `gtouch`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -31,17 +26,16 @@ requirements:
     - pkg-config
     - gperf
     - expat
-    - gettext                     # [not win]
+    - gettext     # [not win]
```
Comment (marcoesters) [inline recipe/meta.yaml]: We are skipping Windows, so we shouldn't need the selector

File: recipe/meta.yaml [hunk 3]
```diff
@@ -31,17 +26,16 @@ requirements:
...
-    - patch       # [not win]
-    - m2-patch    # [win]
- 
+
   host:
     - freetype
     - libxml2
-    - libuuid                     # [linux]
+    - unzip
+    - coreutils  # [osx]
```
Comment (marcoesters) [inline recipe/meta.yaml:40]: Is this for gtouch? If so, please make sure to remove it if we remove the alias.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -31,17 +26,16 @@ requirements:
...
     - autoconf
-    - patch       # [not win]
-    - m2-patch    # [win]
- 
+
   host:
     - freetype
     - libxml2
-    - libuuid                     # [linux]
+    - unzip
```
Comment (marcoesters) [inline recipe/meta.yaml]: I'm not sure it's obvious why this is needed. Maybe add a comment?

File: recipe/test_fix.patch [hunk 5]
```diff
@@ -0,0 +1,44 @@
+diff --git a/test/run-test.sh b/test/run-test.sh
```
Comment (marcoesters) [inline recipe/test_fix.patch]: This patch file is not used.

File: recipe/build.sh (hunk 1 above)
Comment (psteyer) [reply recipe/build.sh]: gtouch was used instead of touch due to failures happening with the native touch command in the tests. @ELundby45 might be able to elaborate more on the why.

Comment (ELundby45) [reply recipe/build.sh]: `touch -d 0 test.test`

This works on most unix systems, but not in MacOS. Adding `coreutils` brings in the GNU compliant version of touch which is what the unit test is expecting. 

It probably makes sense to only set the alias for MacOS.

Comment (marcoesters) [reply recipe/build.sh]: I agree the alias should only be set for MacOS. And we probably can remove the patch, too.

File: recipe/meta.yaml (hunk 2 above)
Comment (psteyer) [reply recipe/meta.yaml]: Removed the selector

**Comment by marcoesters** [comment]
> Linter check found the following problems:
> The following problems have been found:
> 
> ERROR: /tmp/abs_cb00boo1rg/clone/recipe/meta.yaml:45: invalid_url: https://opensource.org/licenses/MIT : Not reachable: 503 Errors were found

It looks like there is a 503 for the `license_url` - since you have a `license_file` already, you can remove this field.

File: recipe/test_fix.patch (hunk 5 above)
Comment (psteyer) [reply recipe/test_fix.patch]: Removed patch files

File: recipe/meta.yaml [hunk 6]
```diff
@@ -1,20 +1,15 @@
...
   name: fontconfig
   version: {{ version }}
 
 source:
-  url: https://www.freedesktop.org/software/fontconfig/release/fontconfig-{{ version }}.tar.bz2
-  sha256: f655dd2a986d7aa97e052261b36aa67b0a64989496361eca8d604e6414006741
-  patches:
-    # This patch won't be submitted upstream. It's use here is to ensure the prefix used
-    # to refer to the `fonts.conf` file.
-    - fcf.patch       # [linux]
```
Comment (cbouss) [inline recipe/meta.yaml:13]: fcf.patch is a change we want to keep.

File: recipe/meta.yaml (hunk 4 above)
Comment (cbouss) [reply recipe/meta.yaml]: same here, why is libuuid removed and unzip added?

**Review by cbouss — COMMENTED** [review]
It would be nice to add a test section.

Conda forge has this:
https://github.com/conda-forge/fontconfig-feedstock/blob/main/recipe/meta.yaml#L43

Ideally we would also test that fontconfig is found using pkg-config. Example:
https://github.com/conda-forge/gobject-introspection-feedstock/blob/main/recipe/meta.yaml#L154

File: recipe/meta.yaml (hunk 4 above)
Comment (psteyer) [reply recipe/meta.yaml]: `unzip` is required for the test `MT-safe global config`, it seems to be downloading a zipped file for the test. I'll add a comment for clarity.

I haven't been able to figure out why `libuuid` is causing the build to fail, but when removed everything seems to be working.

File: recipe/meta.yaml (hunk 3 above)
Comment (psteyer) [reply recipe/meta.yaml:40]: `coreutils` was added for `gtouch`.

File: recipe/build.sh (hunk 1 above)
Comment (psteyer) [reply recipe/build.sh]: went ahead and added some code thanks to @ELundby45 to make it MacOS specific

**Review by marcoesters — APPROVED** [review]
LGTM

**Review by cbouss — COMMENTED** [review]
Can you actually move the test command `test -f ${PREFIX}/lib/pkgconfig/fontconfig.pc` to run_test.sh?
Otherwise run_test.sh won't be executed.
Once done please merge.
